### PR TITLE
allow fast finish on travis testing EDIT: also travis update and documenter cleanup

### PR DESCRIPTION
**Pull request by dehann**

### CODE REVIEW

**Comment by Affie** [comment]
I don't know if its in the correct place now. the build is not failing so can't test.
I'm not exactly sure we want this. I would rather have allow failures only on the release branch and IIF test always done on PRs.

**Comment by dehann** [comment]
oh thanks, didnt think of overwriting!

**Comment by dehann** [comment]
> I would rather have allow failures only on the release branch and IIF test always done on PRs.

we can do that I think.  Shall we just update the allow failures matrix entries?

**Comment by Affie** [comment]
@dehann do you perhaps know how the release testing of julia works? do they actually run you tests. 
Can it somehow cause a problem if the branch is set to not test on release branches?

**Comment by dehann** [comment]
> do you perhaps know how the release testing of julia works? do they actually run you tests.

@Affie, currently Registry bots do not run individual package tests.  There was a time that JuliaCIBot (previous attobot world) would run all package as well as downstream dependencies as a matrix of tests.  The bot would flake out if your changes broke downstream packages on patch releases (ie forcing semver).  Will see if that comes back with new zepto system.

The current public testing only checks for Registry versioning type stuff.  Not sure, but they likely testing for security stuff privately somewhere too.  I'm not sure what is going on "behind the scenes" if there is something like that.  Although part of the assumption is  that public repos are heavily scrutinized by the public anyway.

> Can it somehow cause a problem if the branch is set to not test on release branches?

dont think so, and feel free to try!

File: docs/make.jl [hunk 1]
```diff
@@ -1,5 +1,4 @@
 using Documenter
-using DataFrames
```
Comment (dehann) [inline docs/make.jl]: lol

Comment (dehann) [reply docs/make.jl]: thanks

**Pull request merged**